Modules/Devices/src/Devices.Infrastructure/Persistence/Repository/IdentitiesRepository.cs

```diff
@@ -61,7 +63,7 @@ public async Task<int> CountByClientId(string clientId, CancellationToken cancel
 
     public async Task AddUser(ApplicationUser user, string password)
     {
-        var createUserResult = await _userManager.CreateAsync(user, password);
+        var createUserResult = await _serviceProvider.GetRequiredService<UserManager<ApplicationUser>>().CreateAsync(user, password);
```

Comment:
Why this change?

Reply:
This is a really bad anti pattern called "Service Locator" (see https://stackoverflow.com/questions/22795459/is-servicelocator-an-anti-pattern). We use dependency injection!

Reply:
I looked for alternatives at the time of the commit and I have looked again. I don't see how we can create a dummy of `UserManager<ApplicationUser>` for use in this Job. Any suggestions?